code/game/objects/items/weapons/storage/storage_base.dm

```diff
@@ -447,25 +448,37 @@
 		var/mob/M = _M
 		if((M.s_active == src) && M.client)
 			M.client.screen += I
+	if(user)
+		if(user.client && user.s_active != src)
+			user.client.screen -= I
+		I.dropped(user, TRUE)
+	add_fingerprint(user)
 
-	if(usr)
-		if(usr.client && usr.s_active != src)
-			usr.client.screen -= I
-		I.dropped(usr, TRUE)
-		add_fingerprint(usr)
+	if(!prevent_warning)
+		var/viewer_list = GLOB.player_list
```

Comment:
Also, is player_list a list of mobs, right? And not clients?
```suggestion
		var/viewer_list = GLOB.player_list - user
```

Reply:
Yes, player_list is the list of all mobs with clients attached.